lms/djangoapps/bookmarks/api.py

```diff
@@ -0,0 +1,27 @@
+"""
+This File contains bookmark api method(s).
+"""
+
+from bookmarks.serializers import BookmarkSerializer
+from bookmarks.models import Bookmark
+
+
+def get_bookmark(user, usage_key, fields=None, serialized=False):
```

Comment:
It's only necessary to create a separate api.py file if you expect that in-process callers will want to call "get_bookmark" (without going through the REST api). Otherwise, just keep all the code in the view.

What usage patterns are you thinking of?